modules/client-configuration/pages/clients-debian.adoc

```diff
@@ -165,6 +165,14 @@ mgr-create-bootstrap-repo --with-custom-channels
 For {debian}{nbsp}10, select ``debian10-amd64-uyuni`` when prompted.
 
 
+[WARNING]
+====
+Some clients experience SSL errors when you register them in the {productname} {webui} using a bootstrap script.
```

Comment:
I am not aware of this problem on Debian, not even Debian 9. I think this can go away.